lib/providers/device-app-data-provider.ts

```diff
@@ -76,11 +76,11 @@ export class AndroidNativeScriptCompanionAppIdentifier extends deviceAppDataBase
 	}
 	
 	public get deviceProjectRootPath(): string {
-		return this.getDeviceProjectRootPath(path.join("/data/data", this.appIdentifier, "files"));
+		return this.getDeviceProjectRootPath(path.join("/mnt/sdcard/Android/data", this.appIdentifier, "files"));
```

Comment:
Similar path exists in a livesync-service.ts (DEVICE_TMP_DIR_FORMAT_V3) - you may consider extracting a common constant.